upgradelog/ignition-testnet/consensus_parameters/18.json

```diff
@@ -0,0 +1,305 @@
+{
+  "V2": {
+    "tx_params": {
+      "V1": {
+        "max_inputs": 255,
+        "max_outputs": 255,
+        "max_witnesses": 255,
+        "max_gas_per_tx": 30000000,
+        "max_size": 112640,
```

Comment:
@xgreenx you want to increase this to `2621440` - `112640` to allow space for mint tx?